packages/ui/src/proposals/constants/styles.ts

```diff
@@ -2,7 +2,7 @@ import styled from 'styled-components'
 
 import { ListHeaders, ListHeader } from '@/common/components/List/ListHeader'
 
-export const ProposalColLayout = '296px 148px 156px 1fr'
+export const ProposalColLayout = '296px 148px 156px 60px 92px 35px'
```

Comment:
Nice catch on the voting buttons, I had completely forgot about these, but please keep the last column to `1fr` to match the design alignment on bigger screens:
```suggestion
export const ProposalColLayout = '296px 148px 156px 60px 92px 1fr'
```